app/src/main/java/com/example/parkingsystem_kotlin/mvp/view/ParkingView.kt

```diff
@@ -2,11 +2,19 @@ package com.example.parkingsystem_kotlin.mvp.view
 
 import android.app.Activity
 import android.widget.Toast
+import androidx.fragment.app.DialogFragment
+import com.example.parkingsystem_kotlin.fragment.ParkingSizeDialogFragment
 import com.example.parkingsystem_kotlin.mvp.contract.ParkingContract
 import com.example.parkingsystem_kotlin.mvp.view.base.ActivityView
+import com.example.parkingsystem_kotlin.util.Constants
 
 class ParkingView(activity: Activity): ActivityView(activity), ParkingContract.View {
 
+    override fun showParkingSizeDialogFragment() {
+        val dialogFragment: DialogFragment = ParkingSizeDialogFragment()
+        dialogFragment.show(fragmentManager, Constants.TAG_PARKING_SIZE_DIALOG)
```

Comment:
there's no need to create an extra var yo can call  ParkingSizeDialogFragment().show(...

Reply:
Thanks